### DISCUSSION TITLE
Loading vue apps in a main vue app container

### DISCUSSION THREAD

**Discussion by hoseinmirian** [SAP/luigi#3077] [General]
**Description**

I was wondering if we can change the RTL/LTR of the project along with other customization of the menus, etc..
second I was wondering this case scenario:

I have a main container vue app as hub or host for other micro apps:
I have another repo containing vue app 2
I have another vue containing vue app 3

Now my questions are:
1- is it possible to upload each micro vue app in one server and call them? your answer is probably yes! then if yes I ask how do I communicate the credential and of host app with the micro apps? we know all are vue apps

2- is it possible to load vue app dynamically like a web component like module federation loaded on the same server and reading its URL from the same server? like loading vue main app on one port the other app_1_micro and app_2_micro on another ports and just refer the URL in the config?

**Reasons**

Imagine I ll be having a project I d like to customize it...

**Attachments**

**Comment by hardl** [comment]
The app shell UI components shipped with luigi don't have dedicated RTL support (yet), however you can fully customize the shell ui (https://docs.luigi-project.io/docs/faq?section=ui-questions).

Regarding the second point:
1- yes, you can have all micro frontends hosted on the same server. Communication is generally done via "context" node parameter (https://docs.luigi-project.io/docs/navigation-parameters-reference?section=node-parameters) for the initial loading and/or Luigi API, e.g. custom messages for non-standard communication (https://docs.luigi-project.io/docs/communication/?section=custom-messages).
2- If you stick to luigis functionality only it doesn't matter where your micro frontends are running, be it on the same host, port, or somewhere else.

BR
